tests/tests-core/src/main/java/com/gentics/mesh/core/field/binary/BinaryFieldTest.java

```diff
@@ -287,6 +292,9 @@ public void testRemoveRequiredFieldViaNull() {
 				updateContainer(ac, container, BINARY_FIELD, null);
 			});
 		}
+		tx(tx -> {
```

Comment:
Consistency checks are executed in every test anyways.

Reply:
This is not a check, but repair. This particular test does not use the REST API for the binary manipulation --> this means that in order to please the every-test consistency check, one has to remove the orphaned binary data manually.